templates/security/fips.html

```diff
@@ -12,10 +12,10 @@
    <div class="col-8">
      <h1>FIPS for Ubuntu</h1>
      <h2 class="p-heading--4">FIPS 140 validated cryptography for Linux workloads on Ubuntu</h2>
-     <p>Developing and running Linux workloads for U.S. government regulated and high-security environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 certified cryptographic packages of <a href="/cloud/public-cloud">Ubuntu Pro</a> and <a href="/pro">Ubuntu Advantage</a>.</p>
```

Comment:
The hyphen between "high" and "security" should not be deleted, that is what is on the copy doc.

Reply:
Resolved the comment on the doc since that's already what's live